system/lib/libunwind/test/lit.cfg.py

```diff
@@ -0,0 +1,10 @@
+# All the Lit configuration is handled in the site configs -- this file is only
```

Comment:
Maybe remove the entire "test" directory?

Reply:
Done